static/app/views/replays/detail/eventMetaData.tsx

```diff
@@ -19,16 +20,22 @@ type Props = {
 };
 
 function EventMetaData({crumbs, duration, event}: Props) {
+  const {
+    params: {eventSlug},
+  } = useRouteContext();
+  const {projects} = useProjects();
   const errors = crumbs?.filter(crumb => crumb.type === 'error').length;
 
+  const project = eventSlug.split(':')[0] || '';
+
   return (
     <KeyMetrics>
       <ProjectBadge
-        project={{
-          slug: event?.projectSlug || '',
-          id: event?.projectID,
-          platform: event?.platform as PlatformKey,
-        }}
+        project={
+          projects.find(p => p.slug === project) || {
+            slug: event?.projectSlug || '',
+          }
+        }
         avatarSize={16}
       />
```

Comment:
Showing the ProjectBadge first, before everything else has loaded looks strange to me in the browser. It's the only filled in data when everywhere else there are placeholders:
<img width="1048" alt="Screen Shot 2022-07-25 at 10 52 12 AM" src="https://user-images.githubusercontent.com/187460/180843198-4f4519d0-5457-4b48-8905-0e3ea279d520.png">

We can't even see the "Beta" tag when things are loading, or if there was an error fetching the data :(


An earlier version had `projects.find(p => p.id === event?.projectID)` as the condition... i liked it because it reads from the `event` which is a source of truth, the url can have anything in there.

What we could do to prevent the black icon from appearing is check that the `event` exists instead of relying on the optional chaining. Instead of `event?.projectID` we'd have:
```
return event ? <ProjectBadge ... > : <Placeholder/>
```

Can we iterate on it some more and see if that's a better experience?

Reply:
Here's the result! 
![image](https://user-images.githubusercontent.com/39612839/180872484-e974db3f-4d17-41c8-a254-17cff01664fa.png)